fundamentals/android/auth-anonymous/Notes/app/src/main/java/com/notes/app/model/service/AccountService.kt

```diff
@@ -8,6 +8,7 @@ interface AccountService {
     val currentUserId: String
     fun hasUser(): Boolean
     suspend fun createAnonymousAccount()
+    suspend fun getUserProfile(): User
```

Comment:
Getting a user from Firebase Auth is not an async operation, right? Maybe we don't need to make this a `suspend` function?

Reply:
Removed suspend.